dotcom-rendering/src/web/components/Card/Card.tsx

```diff
@@ -317,13 +317,16 @@ export const Card = ({
 								</Hide>
 							)}
 							{/* Show the card footer in the same column as the headline content */}
-							{!moreThanTwoSubLinks &&
-								renderFooter({ forceVertical: true })}
+							{!moreThanTwoSubLinks ? (
```

Comment:
This is where setting boolean values as booleans, instead of boolean scan be useful to avoid this. Solving the cause of the issues would make the value more predictable in how it behaves

i.e.
```ts
// Currently
const moreThanTwoSubLinks: boolean | 0 | undefined =
		supportingContent?.length && supportingContent.length > 2;

// Predictable
const moreThanTwoSubLinks: boolean =
		supportingContent?.length && supportingContent.length > 2 ? true : false;
```